### DISCUSSION TITLE
[Feature Request]: IDE Support

### DISCUSSION THREAD

**Discussion by booni3** [rawilk/laravel-form-components#52] [Q&A]
### Feature Description

Can you advise if there is a way to get the `<x-input wire:model="model.sku"/>` being recognised in the IDE, the same way that `<x-form-components::inputs.input wire:model="model.sku"/>` does?

I am using PHP Storm & Laravel Idea specifically but this may be relevant to other IDE's also

**Comment by rawilk** [comment]
I use PHPStorm myself, but I'm not sure if there is a way to do this though. If anyone out there knows how, I'd be happy to update the docs to show how to do it.
